torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8656,6 +8668,43 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_out=False,
            supports_forward_ad=True,
            sample_inputs_func=sample_inputs_transpose_swapdims),
+    OpInfo('T',
+           op=lambda x: x.T,
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.half),
+           dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.half),
+           supports_out=False,
+           supports_forward_ad=True,
+           skips=(  # Fails due to the lambda wrapper
```

Comment:
What is the error that you are seeing? It should just skip the lambda for scripting: https://github.com/pytorch/pytorch/blob/master/test/test_ops.py#L720

Reply:
```python
________________________ TestJitMETA.test_variant_consistency_jit_mT_meta_float32 ________________________                                                                                                           
Traceback (most recent call last):                                                                                                                                                                                   
  File "/home/lezcano/git/pytorch/pytorch/test/test_ops.py", line 799, in test_variant_consistency_jit                                                                                                               
    assert tested, "JIT Test does not execute any logic"
AssertionError: JIT Test does not execute any logic
assert False                                         
_______________________ TestJitCUDA.test_variant_consistency_jit_mT_cuda_complex64 _______________________
Traceback (most recent call last):
  File "/home/lezcano/git/pytorch/pytorch/test/test_ops.py", line 799, in test_variant_consistency_jit
    assert tested, "JIT Test does not execute any logic"
AssertionError: JIT Test does not execute any logic
assert False                                         
________________________ TestJitCUDA.test_variant_consistency_jit_mT_cuda_float32 ________________________
Traceback (most recent call last):
  File "/home/lezcano/git/pytorch/pytorch/test/test_ops.py", line 799, in test_variant_consistency_jit
    assert tested, "JIT Test does not execute any logic"
AssertionError: JIT Test does not execute any logic
assert False  
```